templating/twig_extension.rst

```diff
@@ -98,26 +98,30 @@ callable defined in ``getFilters()``::
     namespace App\Twig;
 
     use App\Twig\AppRuntime;
+    use Twig\Extension\AbstractExtension;
+    use Twig\TwigFilter;
 
-    class AppExtension extends \Twig_Extension
+    class AppExtension extends AbstractExtension
     {
         public function getFilters()
         {
             return array(
                 // the logic of this filter is now implemented in a different class
-                new \Twig_SimpleFilter('price', array(AppRuntime::class, 'priceFilter')),
+                new TwigFilter('price', array(AppRuntime::class, 'priceFilter')),
             );
         }
     }
 
 Then, create the new ``AppRuntime`` class (it's not required but these classes
-are suffixed with ``Runtime`` by convention) and include the logic of the
+are suffixed with ``Runtime`` by convention), implement the ``RuntimeExtensionInterface`` and include the logic of the
```

Comment:
Please move this change in a dedicated PR targeting 3.4 and rebase the current on 2.8. Thanks!